index.html

```diff
@@ -0,0 +1,26 @@
+<head>
+  <link rel="stylesheet" href="./style.css" type="text/css">
+  <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+</head>
+<body>
+<div>
+  <h3 class="">Simon</h3>
+  <div class="row">
+    <div>
+      <button class="btn btn-default target start clickable" id="Start">Start</button>
+    </div>
+
+  </div>
+  <div class="buttons" id="buttons">
+    <div class="row">
+      <button class="btn btn-default clickable target green" id="Green">Green</button>
+      <button class="btn btn-default clickable target red" id="Red">Red</button>
+    </div>
+    <div class="row">
+      <button class="btn btn-default clickable target yellow" id="Yellow">Yellow</button>
+      <button class="btn btn-default clickable target blue" id="Blue">Blue</button>
```

Comment:
The classes `btn` and `btn-default` don't seem to be being used.  Do you plan to use them in the future?